admin/builders/centos64_build.sh

```
@@ -57,61 +55,20 @@ else
 (set -x; \
-cmake -DTARGET_ARCH=x86_64 -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_PREFIX_PATH="${build_dir}/${centos_boost}" -DSERIALIZE="TokyoCabinet" ${src_dir}/percolator/src/converters; \
+cmake -DTARGET_ARCH=x86_64 -DCMAKE_BUILD_TYPE=Release -DGOOGLE_TEST=1 -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_PREFIX_PATH="${build_dir}/${centos_boost}" ${src_dir}/percolator; \
 )
 make -j 4
+make test
 make -j 4 package
 
 echo "build directory was : ${build_dir}"
 
-cp -v ${build_dir}/{percolator-noxml,percolator,converters}/*.rpm ${release_dir}
+cp -v ${build_dir}/percolator*.rpm ${release_dir}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)